### PR TITLE
feat: Configure Images portlet into news page template - EXO-65514 - Meeds-io/MIPs#68

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

**Review by azayati — APPROVED** [review]
LGTM

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml [hunk 1]
```diff
@@ -31,9 +31,14 @@
         template="system:/groovy/portal/webui/container/UIContainer.gtmpl"
         cssClass="border-box-sizing">
         <access-permissions>Everyone</access-permissions>
-        <container id="topNewsPage" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
-          <access-permissions>*:/platform/users</access-permissions>
-        </container>
+        <portlet-application>
```
Comment (azayati) [inline webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: I think that we should rework the news template layout, to be verified with the PO

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml [hunk 2]
```diff
@@ -31,9 +31,14 @@
...
+          <portlet>
+            <application-ref>social-portlet</application-ref>
+            <portlet-ref>ImagesPortlet</portlet-ref>
+          </portlet>
+          <access-permissions>Everyone</access-permissions>
+          <show-info-bar>false</show-info-bar>
+        </portlet-application>
         <container id="newsBodyPage" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
```
Comment (azayati) [inline webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]:
```suggestion
        <container id="newsBodyPage" template="system:/groovy/portal/webui/container/UIResponsiveColumnGroupContainer.gtmpl.gtmpl">
```

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml [hunk 3]
```diff
@@ -31,9 +31,14 @@
...
           <access-permissions>Everyone</access-permissions>
           <container cssClass="newsBodyPageLeftColumn"
```
Comment (azayati) [inline webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: Change newsBodyPageRightColumn and newsBodyPageLeftColumn from UIContainer.gtmpl to UISimpleColumnContainer.gtmpl

In that case I think that added [css classes](https://github.com/exoplatform/news/commit/af3952fe2d05f25d7c2dde134340c67ecd32a3cd#diff-77f3cfb6a6caf540496eab73a1012e948b54b7b600c2510c14e9298f93a13ba7) are no more needed

**Review by azayati — CHANGES_REQUESTED** [review]

File: webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml (hunk 1 above)
Comment (azayati) [reply webapp/src/main/webapp/WEB-INF/conf/portal/template/pages/news/page.xml]: I think that the container id="newsPage" is nomore needed, even css classes already added not needed, to be verified with the final look

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**